BlueprintUI/Sources/Internal/ElementIdentifier.swift

```diff
@@ -41,24 +41,24 @@
 
  You will note that the identifiers remain stable, which ultimately ensures that views are reused.
  */
-struct ElementIdentifier: Hashable, CustomDebugStringConvertible {
+struct ElementIdentifier: Hashable, CustomStringConvertible {
```

Comment:
Noting for later: Another win we might get is [adopting the `ElementIdentifier` caching](https://github.com/square/Blueprint/pull/398/files#diff-44612c52b487b560a6d4dadecd96df4cd9e7c4a53a7f9c9f5b2b933a9700649f) from the caching branch. It's likely not as valuable here given the measurement caches themselves don't leverage Element ID as heavily, but it likely still provides some benefit.